ros-base/Dockerfile

```diff
@@ -40,6 +41,11 @@ RUN source /opt/ros/${ROS_DISTRO}/setup.bash \
     # Build catkin workspace
     && catkin_make
 
+# Always source ros_entrypoint.sh when launching bash (e.g. when attaching to container)
+RUN echo "source /ros_entrypoint.sh" >> /root/.bashrc \
+    # Make rest of shell session set +e so bash isn't killed on every non-zero return code
+    && echo "set +e" >> /ros_entrypoint.sh # Reset e after 
```

Comment:
Rather than modifying this here, wouldn't it make more sense to change `set +e` on `ros_catkin_entrypoint.sh`?

Reply:
Good point. I somehow thought set -e was from upstream.

We could also subshell it if we want, but then I think we lose stdout and stderr. https://stackoverflow.com/questions/24460504/bash-stop-on-error-in-sourced-script